test/inspector/inspector-helper.js

```diff
@@ -68,7 +68,7 @@ function parseWSFrame(buffer, handler) {
     dataLen = buffer.readUInt16BE(2);
     bodyOffset = 4;
   } else if (dataLen === 127) {
-    dataLen = buffer.readUInt32BE(2);
+    dataLen = buffer.readUIntBE(2, 8);
```

Comment:
I guess it's safe to assume that frames are not bigger than `2 ** 32 -1` (I'm not sure how they are generated). In that case we can ignore the first 4 bytes:

```js
dataLen = buffer.readUInt32BE(6);
```